apps/omg_utils/lib/omg_utils/remote_ip.ex

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+defmodule OMG.Utils.RemoteIP do
```

Comment:
```
If a request goes through multiple proxies, the IP addresses of each successive proxy is listed. This means, the right-most IP address is the IP address of the most recent proxy and the left-most IP address is the IP address of the originating client.

```

is this covered by this plug? is it an issue @arthurk ?

Reply:
nope, it's not covered. but I can add if it's needed

Reply:
yeah that seems right it's pretty normal to have multiple IPs there especially when using proxies such as Cloudflare. 

I think cowlib has some functionality related to this (which should already be pulled in as part of cowboy?):

https://github.com/ninenines/cowlib/blob/2ecd128438f9b5b4d9979e89113ef1a9d47acdb7/src/cow_http_hd.erl#L3551

Reply:
Now I set the left most ip